jax/experimental/jax_to_tf/tests/tf_test_util.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Callable, Sequence, Tuple
+import tensorflow as tf  # type: ignore[import]
+
+from jax.experimental import jax_to_tf
+from jax import test_util as jtu
+
+class JaxToTfTestCase(jtu.JaxTestCase):
+
+  def assertAllClose(self, x, y, *, atol=None, rtol=None,
+                     check_dtypes=False, canonicalize_dtypes=False):
+    """Compares recursively list/tupls of arrays."""
+    # TODO: turn on check_dtypes (get error comparing TF dtype and JAX dtype)
```

Comment:
I'd suggest asserting that the kwargs here have their default value as well as the comment.

Reply:
I did not quite understand the assert suggestion. I was planning to address the dtype 
comparison across numpy and TF later, but it was actually easy to do.